plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/AbstractDeltaLakePageSink.java

```diff
@@ -361,7 +361,7 @@ private int[] getWriterIndexes(Page page)
                 partitionName = Optional.of(partName);
             }
 
-            String fileName = session.getQueryId() + "-" + randomUUID();
+            String fileName = session.getQueryId() + "_" + randomUUID();
```

Comment:
This change is in sync with https://github.com/trinodb/trino/blob/20e57792d27c2d98d758332978f16f0dc94baa28/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java#L333

I'd benefit from a suggestion in which utility class could i extract the method for creating the file name.

Reply:
You can always add a new one ;)